slash/utils/cli_utils.py

```diff
@@ -167,6 +167,23 @@ def make_styler(style):
 UNDERLINED = '\x1b[4m'
 
 
+class Printer(object):
+
+    def __init__(self, report_stread, disable_output=False):
+        self._stream = report_stread
+        self._disabled = disable_output
+        self._func = self._no_op if disable_output else self._colored_print
+
+    def _no_op(self, *args):
```

Comment:
This is an overkill IMO. It's better for `_colored_print` itself to check for enabled-ness